apps/api-extractor/src/generators/packageTypings/SymbolAnalyzer.ts

```diff
@@ -93,11 +93,30 @@ export class SymbolAnalyzer {
       current = currentAlias;
     }
 
+    // Is the followedSymbol actually the kind of thing that can be ambient?
+    if (isAmbient) {
+      for (const declaration of current.declarations || []) {
+        switch (declaration.kind) {
+          case ts.SyntaxKind.ClassDeclaration:
+          case ts.SyntaxKind.InterfaceDeclaration:
+          case ts.SyntaxKind.FunctionDeclaration:
+          case ts.SyntaxKind.ModuleDeclaration:
+          case ts.SyntaxKind.VariableDeclaration:
```

Comment:
this is a subset of the things in the other file

Reply:
I moved both switch statements into SymbolAnalyzer for now.

---
In reply to: [174002883](https://github.com/Microsoft/web-build-tools/pull/560#discussion_r174002883) [](ancestors = 174002883)